examples/post-process/bloom.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Unreal Bloom Effect
+ * Implementation for A-Frame
+ * Code modified from Akbartus's UnrealBloomPass.js
+ * https://github.com/akbartus/A-Frame-Component-Postprocessing
+ */
+
+import AFRAME from "aframe";
+import { EffectComposer } from "three/addons/postprocessing/EffectComposer.js"; // This uses the same three instance.
+import { RenderPass } from "three/addons/postprocessing/RenderPass.js"; // This uses the same three instance.
+import { UnrealBloomPass } from "three/addons/postprocessing/UnrealBloomPass.js"; // This uses the same three instance.
```

Comment:
You can remove the comments. I put the comments on the importmap example to explain that it fixes the multiple instance of three warning, that but those comments shouldn't be put on other examples.